types/koa2-cors/index.d.ts

```diff
@@ -0,0 +1,21 @@
+// Type definitions for koa2-cors 2.0
+// Project: https://github.com/zadzbw/koa2-cors#readme
+// Definitions by: xialeistudio <https://github.com/xialeistudio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+import * as Koa from 'koa';
+declare namespace cors {
+    interface Options {
+        origin?: any;
```

Comment:
The type can better be `string | (ctx: Koa.Context) => string` https://github.com/zadzbw/koa2-cors#origin

Reply:
But it can be`boolean`,that make me confounded.
do you have any ideas?

Reply:
The document says: `Configures the Access-Control-Allow-Origin CORS header. expects a string. Can also be set to a function, which takes the ctx as the first parameter.`
The source sode says this too: https://github.com/zadzbw/koa2-cors/blob/5c38ea409ab3cb6b73177012bda26ef81fd4d874/src/index.js#L5

Reply:
ok.I see,wait a moment